css/style.css

```diff
@@ -0,0 +1,3294 @@
+    font-size: 7px;
+    width: 59.67px;
+  }
+
+  .icon-security__rollOver {
+    font-size: 7px;
+    width: 80.37px;
+  }
+}
+
+@media screen and (min-width: 321px) and (max-width: 375px) {
+  .header {
+    max-width: 375px;
+    min-width: auto;
+    padding: 5px 21px 30px 0;
+    position: static;
+  }
+
+  .header__nav {
+    display: none;
+  }
+
+  .header__menu-list {
+    display: none;
+  }
+
+  .characteristic {
+    max-width: 375px;
+    min-width: auto;
+    padding: 0;
+  }
+
+  .characteristic__title {
+    font-size: 50px;
+  }
+
+  .characteristic__text {
+    font-size: 10px;
+  }
+
+  .characteristic__logoImage {
+    padding-top: 70px;
+    width: 100%;
+  }
+
+  .characteristic__carImage {
+    max-width: 100%;
+    padding-top: 40px;
+  }
+
+  .characteristic__description {
+    max-width: 100%;
+    padding: 18px 40px 43px;
+    display: flex;
+    flex-direction: row;
+    box-sizing: border-box;
+  }
+
+  .description-characteristic__speedo {
+    padding-right: 41px;
+  }
+
+  .speedo__image {
+    width: 38.14px;
+    height: 27.98px;
+  }
+
+  .speedo__title {
+    margin-top: -15px;
+  }
+
+  .speedo__title-numb {
+    font-size: 27px;
+  }
+
+  .speedo__title-text {
+    font-size: 15px;
+  }
+
+  .speedo__desc {
+    font-size: 10px;
+  }
+
+  .description-characteristic__road {
+    padding-right: 0;
+  }
+
+  .road__image-stroke {
+    width: 24.04px;
+    height: 36.88px;
+    padding-top: 10px;
+  }
+
+  .road__image-strokeDotted {
+    width: 21.89px;
+    height: 34.74px;
+    top: 18px;
+  }
+
+  .road__image-ellipseRightTop {
+    width: 5.8px;
+    height: 5.74px;
+    top: 16px;
+  }
+
+  .road__image-ellipseRightTopIn {
+    display: none;
+  }
+
+  .road__image-ellipseLeftBottom {
+    top: 49px;
+  }
+
+  .road__title-numb {
+    font-size: 27px;
+    padding-top: 13px;
+  }
+
+  .road__title-text {
+    font-size: 15px;
+  }
+
+  .road__desc {
+    font-size: 10px;
+  }
+
+  .description-characteristic__testDrive {
+    display: none;
+  }
+
+  .description-characteristic__button {
+    display: none;
+  }
+
+  .description-characteristic__wrap-phone {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .description-characteristic__testDrive-phone {
+    display: flex;
+    align-items: center;
+    padding-right: 20px;
+  }
+
+  .testDrive-phone__image {
+    position: relative;
+    padding-right: 40px;
+  }
+
+  .testDrive-phone__image-ellipse {
+    position: absolute;
+    width: 22px;
+    height: 22px;
+    top: -11px;
+    left: 10px;
+  }
+
+  .testDrive-phone__image-tesla {
+    position: absolute;
+    width: 14.18px;
+    height: 8.23px;
+    left: 14px;
+    top: -1px;
+    animation-duration: 3s;
+    animation-name: rotatePhone;
+    animation-iteration-count: infinite;
+    animation-direction: alternate;
+    transform-origin: 50% 0%;
+  }
+
+  @keyframes rotatePhone {
+    from {
+      transform: rotate(0deg);
+    }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/tesla_project/pullRequest?prid=5586613)